datafusion/sql/src/expr/value.rs

```diff
@@ -196,127 +195,42 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
             );
         }
 
-        // Only handle string exprs for now
```

Comment:
It is great to remove all this stuff from the sql planner (as it is now in the parser)